src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/Example2.java

```diff
@@ -8,10 +8,10 @@
 </module>
 */
 
-// xdoc section -- start
-package com.puppycrawl.tools.checkstyle.checks;  //No Violation
 
-public class AnnotationLocationCheck extends AbstractCheck {
-  //...
+// xdoc section -- start
+package checkstyle.coding.directory.someotherpackage;
```

Comment:
To avoid spelling, type word "nonexistent"

Reply:
Done